ang/civicase-base/providers/activity-actions.provider.js

```diff
@@ -0,0 +1,56 @@
+(function (angular, $, _) {
+  var module = angular.module('civicase-base');
+
+  module.provider('ActivityActions', function () {
+    var actions = [{
+      title: 'Resume Draft',
+      icon: 'play_circle_filled',
+      name: 'ResumeDraft'
+    }, {
+      title: 'View in Feed',
+      icon: 'pageview',
+      name: 'ViewInFeed'
+    }, {
+      title: 'Edit',
+      icon: 'edit',
+      name: 'Edit'
+    }, {
+      title: 'Print Report',
+      icon: 'print',
+      name: 'PrintReport'
+    }, {
+      title: 'Move to Case',
+      icon: 'next_week',
+      name: 'MoveCopy',
+      operation: 'move'
+    }, {
+      title: 'Copy to Case',
+      icon: 'filter_none',
+      name: 'MoveCopy',
+      operation: 'copy'
+    }, {
+      title: 'Tag - add to activities',
+      icon: 'add_circle',
+      name: 'Tags',
+      operation: 'add'
+    }, {
+      title: 'Tag - remove from activities',
+      icon: 'remove_circle',
+      name: 'Tags',
+      operation: 'remove'
+    }, {
+      title: 'Download All',
+      icon: 'file_download',
+      name: 'DownloadAll'
+    }, {
+      showDividerBeforeThisAction: true,
+      title: 'Delete',
+      icon: 'delete',
+      name: 'Delete'
+    }];
```

Comment:
Can we add the full service name here? We could define each item similar to this one:

```js
{
  title: 'Download All',
  icon: 'file_download',
  serviceName: 'DownloadAllActivityAction'
}
```

The reason for this is that it's easier to know the full service name. Right now we need to look at this list and then see where it's being used and then find the `getActionService` method to figure out that the code for this action is in `DownloadAllActivityAction`.

Bonus point for this is that we could get rid of the local `getActionService` and just have a generic `getService` since we are using this pattern more often.

Reply:
I initially tried this, but the problem is with the Actions coming from Backend. For example `View Activity`. There we cant have `serviceName`. Hence I am using `name` as a identifier.